build.gradle

```diff
@@ -36,6 +36,8 @@ jenkinsPlugin {
   // Plugin ID, defaults to the project name without trailing '-plugin'
   shortName = 'gradle'
 
+  fileExtension = 'jpi'
```

Comment:
was there a reason for this change it appears to have broken the Jenkins acceptance test harness tests.

see https://github.com/jenkinsci/acceptance-test-harness/issues/985